java/HelperMethods.java

```diff
@@ -1,7 +1,37 @@
 // The tweet that started it all: https://twitter.com/nilsding/status/740241674597761024
 
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.List;
+import java.util.ArrayList;
+
 public class HelperMethods {
 
+    public static int[] sleepSort(int... args) {
```

Comment:
Place new methods at the bottom of the class.

Reply:
![image](https://user-images.githubusercontent.com/8442384/33958189-07760ca4-e044-11e7-92ca-c59e6a068751.png)